src/api/spec/features/beta/webui/requests_index_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'browser_helper'
+
+RSpec.describe 'Requests Index' do
+  let(:submitter) { create(:confirmed_user, :with_home, login: 'kugelblitz') }
+  let(:receiver) { create(:confirmed_user, :with_home, login: 'titan') }
+  let(:target_project) { receiver.home_project }
+  let(:target_package) { create(:package, name: 'goal', project_id: target_project.id) }
+  let(:source_project) { submitter.home_project }
+  let(:source_package) { create(:package, name: 'ball', project_id: source_project.id) }
+
+  let!(:bs_requests) do
+    create(:bs_request_with_submit_action,
+           description: 'Please take this',
+           creator: submitter,
+           target_package: target_package,
+           source_package: source_package)
+
+    create(:bs_request,
+           type: 'maintenance_release',
+           source_project: submitter.home_project,
+           creator: submitter)
+  end
+
+  context 'request index page' do
+    describe 'show all requests' do
+      before do
+        login submitter
+        Flipper.enable(:request_index)
+        visit requests_path(params: { involvement: 'all' })
+      end
```

Comment:
With the help of some `let`s you could DRY this `before` block to be common to all contexts

Reply:
Like this https://github.com/openSUSE/open-build-service/pull/16536/commits/ed4963100b3c4a98678f76e1292ee667a4d45db4 ?